src/librbd/image/RefreshParentRequest.h

```diff
@@ -4,7 +4,6 @@
 #ifndef CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 #define CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 
-#include "include/int_types.h"
```

Comment:
@dillaman since as we know include/int_types.h is so basic header file, and on librbd level(image parent level), there is librbd/parent_types.h, and this has included int_types.h, so there is really no need to reinclude.